File: Plustrack_android/app/src/main/java/com/example/plustrack_android/Employee.kt
package com.example.plustrack_android

import java.util.Date

data class Employee(
    val id: String,
    val name: String,
    val surnames: String,
    val dni: String,
    val birthDate: Date,
    val email: String,
    val password: String,
    val companyId: String? = null,
)

File: PlusTrackForms/PlusTrackForms/PlusTrackForms/Models/Entities/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PlusTrackForms.Models.Entities
{
    public class Employee
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Surnames { get; set; }
        public string Dni { get; set; }
        public DateTime BirthDate { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string companyId { get; set; }
    }
}
